### DISCUSSION TITLE
Release notification spamming

### DISCUSSION THREAD

**Discussion by toby63** [LizardByte/community#454] [Sunshine] [answered]
Sry to bother, but I subscribed to the releases of this repo and for the last two days I got several Emails with pre-release notifications.
I only wanted to ask whether this is intentional and whether it would be possible to handle this a bit differently.

**Comment by ReenigneArcher** [comment, marked as answer]
I apologize for the inconvenience, but it is intentional. This is how the new nightly is going to be published.

If you want only stable notifications you can get those in several places:

- Sunshine UI
- LizardByte or Moonlight Discord announcement channels
- LizardByte subreddit
- LizardByte twitter
- LizardByte facebook

And disable emails from GitHub, only using UI notifications. I don't get any pre-release notifications in the UI.

**Reply by toby63** [reply]
Well thats a bit unfortunate.
But thx for the answer and info.

I understand that you want to publish binaries for testing etc., but maybe a seperate repo could be used for that, something like "Sunshine-Nightly"?
This would also serve bug reports etc. perfectly, because it would seperate the nightly bug reports from the rest.

**Reply by toby63** [reply]
For those interested, some people seem to have requested a solution from GitHub's side as well:
https://github.com/orgs/community/discussions/109626

**Reply by ReenigneArcher** [reply]
Here's the original request: https://github.com/orgs/community/discussions/4993

**Reply by ReenigneArcher** [reply]
> I understand that you want to publish binaries for testing etc., but maybe a seperate repo could be used for that, something like "Sunshine-Nightly"?
This would also serve bug reports etc. perfectly, because it would seperate the nightly bug reports from the rest.

The change was made to simplify maintenance of the project (and other repos in the LizardByte org). Other than Sunshine, I am maintaining most of the 40+ repos by myself, and Sunshine has 3-5 people, which is not enough to keep up with the user demand on the repo... The project is quite overwhelming to maintain and the new approach greatly simplifies things for us. The main thing is we can very easy convert a pre-release to a stable release when we feel it's ready, instead of going through multiple more PRs to get a release published.

I guess my suggestion is to disable GitHub emails and use the UI notifications instead, if you regularly login to GitHub. I am pretty sure (but not 100% sure) I do not get pre-release notifications in GitHub's UI. For example, I am subscribed to releases on https://github.com/unknownskl/greenlight and do not remember getting pre-release notifications.

![image](https://github.com/LizardByte/Sunshine/assets/42013603/d5b9abd5-444c-430a-9162-5cdff1b80ac6)

**Reply by toby63** [reply]
Ok, I understand. I will disable notifications for this repo then. 

I also see this mainly as a GitHub problem; still it is no option for me to use other commercial services you recommended for update notifications.
As your software is not included in most official package repos, it is necessary for me to keep track of updates.
But as you seem to maintain the main release on [AUR](https://aur.archlinux.org/packages/sunshine), I will subscribe to that as an alternative.

**Reply by Kissaki** [reply]
It's baffling GitHub has not implemented separate notifications yet.

Guess I'll unsubscribe from releases too then.

Another project created a separate repo just for pre-releases so it wouldn't noise the main repo. It's a bit unfortunate, but would allow to still use stable release notifications. (Which I suspect is the main use case for almost all people for release notifications.)

[a comment by an automated account is not shown]
